src/main/java/uk/gov/justice/hmpps/prison/aop/ProxyUserAspect.java

```diff
@@ -30,18 +30,18 @@ public void proxyUserPointcut() {
     @Around("proxyUserPointcut()")
     public Object controllerCall(final ProceedingJoinPoint joinPoint) throws Throwable {
 
-        var proxyUser = authenticationFacade.getCurrentPrincipal();
+        var authentication = authenticationFacade.getAuthenticationOrNull();
```

Comment:
in the new holder `authentication` can be null if no authentication for call so check that instead